mLRS/Common/hal/esp/tx-hal-jumper-int-2400-esp32.h

```diff
@@ -0,0 +1,175 @@
+//*******************************************************
+// Copyright (c) MLRS project
+// GPL3
+// https://www.gnu.org/licenses/gpl-3.0.de.html
+//*******************************************************
+// hal
+//*******************************************************
+
+//-------------------------------------------------------
+// ESP32, Jumper T20 Internal ELRS
+//-------------------------------------------------------
+
+// https://github.com/ExpressLRS/targets/blob/master/TX/Jumper%20T-20%202400.json
+// Added fan on GPIO 33
+
+#define DEVICE_HAS_JRPIN5_FULL_DUPLEX
+#define DEVICE_HAS_SINGLE_LED
+#define DEVICE_HAS_NO_COM
+#define DEVICE_HAS_NO_DEBUG
+#define DEVICE_HAS_FAN_ONOFF
+#define DEVICE_HAS_ESP_WIFI_BRIDGE_ON_SERIAL
+
+
+//-- UARTS
+// UARTB = serial port to backpack WiFi bridge
```

Comment:
pl add the complete list, see e.g here: https://github.com/olliw42/mLRS/blob/main/mLRS/Common/hal/esp/tx-hal-radiomaster-int-2400-esp32.h#L72-L77